doi_to_org.py

```diff
@@ -378,5 +374,10 @@ def org_entry_fabric(orgfile, pdfpath, arg):
     if CONFIG_PDFPATH_KEY in config:
         pdfpath = config[CONFIG_PDFPATH_KEY]
     for arg in sys.argv[1:]:
-        for entry in org_entry_fabric(orgfile, pdfpath, arg):
-            entry.add_entry()
+        try:
+            for entry in org_entry_fabric(orgfile, pdfpath, arg):
+                entry.add_entry()
+        except FileNotFoundError as e:
+            print(e)
+            sys.exit()
+
```

Comment:
This would be better:
```python
sys.exit(e)
```
It prints the error message on stderr instead of stdout and it is one line less.